src/Spryker/Client/RabbitMq/Model/Connection/Connection.php

```diff
@@ -39,17 +39,17 @@ class Connection implements ConnectionInterface
     /**
      * @param \PhpAmqpLib\Connection\AMQPStreamConnection $streamConnection
      * @param \Spryker\Client\RabbitMq\Model\Helper\QueueEstablishmentHelperInterface $queueEstablishmentHelper
-     * @param \ArrayObject $queueOptionCollection
+     * @param \Generated\Shared\Transfer\QueueConnectionTransfer $queueConnection
      */
     public function __construct(
         AMQPStreamConnection $streamConnection,
         QueueEstablishmentHelperInterface $queueEstablishmentHelper,
-        ArrayObject $queueOptionCollection
+        QueueConnectionTransfer $queueConnection
```

Comment:
`queueConnectionTransfer`

Reply:
Btw why use transfer for this? for configuration i would expect something immutable.